build.gradle

```diff
@@ -264,6 +264,15 @@ test {
         }
     }
 
+    maxHeapSize = "8G"
+    forkEvery = 50
```

Comment:
We actually have another `forkEvery 100` further down in this file, which you might want to remove now